app/indexers/image_indexer.rb

```diff
@@ -0,0 +1,19 @@
+# Generated via
+#  `rails generate hyrax:work Image`
+class ImageIndexer < Hyrax::WorkIndexer
+  # This indexes the default metadata. You can remove it if you want to
+  # provide your own metadata and indexing.
+  include Hyrax::IndexesBasicMetadata
+
+  # Fetch remote labels for based_near. You can remove this if you don't want
+  # this behavior
+  include Hyrax::IndexesLinkedMetadata
+
+
```

Comment:
This line causes CI to fail on (I think) each of these PRs after rebase on the cop-ci branch. How do we feel about removing the line in the rebase?